lib/ControlledVocab/ControlledVocab.js

```diff
@@ -499,7 +534,11 @@ class ControlledVocab extends React.Component {
           <ConfirmationModal
             id="delete-controlled-vocab-entry-confirmation"
             open={this.state.showConfirmDialog}
-            heading={<FormattedMessage id="stripes-core.button.deleteEntry" values={{ entry: type }} />}
+            heading={
+              deleteEntry ?
+                <FormattedMessage id={deleteEntry} /> :
```

Comment:
should we provide `values` prop here?

Reply:
I think it is not needed when using the translations prop.

Reply:
I don't think we need the `values` prop any longer because `deleteEntry` can correspond to sth like `Delete patron group` or `Delete material type`. 

Ofc, we do still need it for `stripes-core.button.deleteEntry` because it looks like `Delete {entry}`.